modules/kafka/src/main/java/com/spotify/ffwd/kafka/KafkaPartitioner.java

```diff
@@ -37,13 +38,18 @@
     int partition(final Metric metric);
 
     class Host implements KafkaPartitioner {
+        private final CRC32 crcGenerator;
+
         @JsonCreator
         public Host() {
+            crcGenerator = new CRC32();
         }
 
         @Override
         public int partition(final Event event) {
```

Comment:
Since this is called by multiple threads (I checked now), we can't use a single instance of CRC32, or each thread will race with each other to update/getValue(), possibly causing inconsistent values.

This should have:
```java
final CRC32 crcGenerator = new CRC32();
```

In the method.